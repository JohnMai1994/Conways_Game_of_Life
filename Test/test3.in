new 5
init 
0 0 0 1 0 2
-1 -1
print
step
print
step
print

