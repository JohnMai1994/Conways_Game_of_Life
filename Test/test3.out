X____
X____
X____
_____
_____
_____
XX___
_____
_____
_____
_____
_____
_____
_____
_____
